### PR TITLE
States in search

### PR DESCRIPTION
**Pull request by sabrina-ulbrich**
States are added to the search and are shown on the result page (new elasticsearch indexing is required). Looks like:
Screenshot 2021-11-17 at 15 36 11

### CODE REVIEW

**Comment by sabrina-ulbrich** [comment]
The last commit (elastic search min score) is necessary otherwise the behavior of the search result page on staging and locally is different.

**Comment by sabrina-ulbrich** [comment]
last fix ;) ready for merging, I think.

File: app/controllers/profiles_controller.rb [hunk 1]
```diff
@@ -198,7 +199,9 @@ def profiles_for_search
       .includes(:taggings, :translations)
       .search(
         params[:search],
+        # params[:filter_countries] || current_region,
```
Comment (zaziemo) [inline app/controllers/profiles_controller.rb:202]: seems you forgot to remove the comment?

File: app/models/concerns/searchable.rb [hunk 2]
```diff
@@ -76,12 +84,13 @@ def self.search(query, filter_countries, filter_cities, filter_lang, with_explai
           }
         }
 
-      query_hash[:min_score] = 3.00 if Rails.env.production?
+      query_hash[:min_score] = 3.00 unless Rails.env.test?
```
Comment (zaziemo) [inline app/models/concerns/searchable.rb:87]: good fix! Absolutely right, we only don't want it in the test environment.

File: app/models/concerns/searchable.rb [hunk 3]
```diff
@@ -95,7 +104,8 @@ def as_indexed_json(_options = {})
 
       output = as_json(
         autocomplete: { input: [fullname, twitter_de, twitter_en, topic_list] },
-        only: %i[firstname lastname iso_languages country],
+        only: %i[firstname lastname iso_languages country state],
+        #only: %i[firstname lastname iso_languages],
```
Comment (zaziemo) [inline app/models/concerns/searchable.rb:108]: do we still need this comment?

File: app/views/shared/_search_filter.erb [hunk 4]
```diff
@@ -13,7 +13,7 @@
             <span class="pl-1"><%# filter[:aggregations].first["doc_count"] %></span>
           </li>
           <% filter[:aggregations][0..9].each do |term| %>
-          <%# binding.pry %>
```
Comment (zaziemo) [inline app/views/shared/_search_filter.erb:16]: both lines should be removed

**Comment by sabrina-ulbrich** [comment]
elasticsearch indexing: bundle exec rake elasticsearch:import:all

**Review by zaziemo — COMMENTED** [review]
only some comments

**Pull request merged**